build.sbt

```diff
@@ -2,7 +2,7 @@ organization := "com.fortysevendeg"
 
 name := "github4s"
 
-version := "0.1"
+version := "0.1-SNAPSHOT"
```

Comment:
We should create  `0.1` version and later we can create a new PR with `0.1-SNAPSHOT`